postgres/datadog_checks/postgres/statement_samples.py

```diff
@@ -361,7 +361,7 @@ def _run_explain_safe(self, dbname, statement, obfuscated_statement):
                 tags=self._dbtags(dbname, "error:explain-{}".format(type(e))) + self._check._get_debug_tags(),
                 hostname=self._check.resolved_hostname,
             )
-            return None, DBExplainError.database_error, '{}'.format(type(err))
+            return None, DBExplainError.database_error, '{}'.format(type(e))
```

Comment:
Is there a test we can add that would have caught this?

Reply:
We would have to find a way to reproduce the error that would fall back to this `except` block (e.g the `undefinedparameter` error we discussed offline) and I don't know the steps to do it off the top of my head. That said, I could definitely look into it, although it may be tricky. We do have some tests that capture some of the obvious/most common collection_errors: https://github.com/DataDog/integrations-core/blob/master/postgres/tests/test_statements.py#L311-L346